### PR TITLE
add on existing ip address feature

### PR DESCRIPTION
**Pull request by romaric1**
HI ,
My pull request resolve this issue : Support reserved IP address #25 
Thanks

### CODE REVIEW

**Comment by romaric1** [comment]
I signed it!

**Comment by romaric1** [comment]
I signed it!

File: main.tf [hunk 1]
```diff
@@ -19,7 +19,7 @@ resource "google_compute_global_forwarding_rule" "http" {
   count      = "${var.http_forward ? 1 : 0}"
   name       = "${var.name}"
   target     = "${google_compute_target_http_proxy.default.self_link}"
-  ip_address = "${google_compute_global_address.default.address}"
+  ip_address = "${var.ip_address}"
```
Comment (tpoindessous) [inline main.tf]: Hi

I think you forgot to handle the case when we don't provide one IP ;) 

```suggestion
  ip_address = "${var.create_ip_address ?  google_compute_global_address.default.address : var.ip_address}"
```

Thanks !

File: main.tf [hunk 2]
```diff
@@ -29,13 +29,14 @@ resource "google_compute_global_forwarding_rule" "https" {
   count      = "${var.ssl ? 1 : 0}"
   name       = "${var.name}-https"
   target     = "${google_compute_target_https_proxy.default.self_link}"
-  ip_address = "${google_compute_global_address.default.address}"
+  ip_address = "${var.ip_address}"
```
Comment (tpoindessous) [inline main.tf]: Hi

I think you forgot to handle the case when we don't provide one IP ;) 

```suggestion
  ip_address = "${var.create_ip_address ?  google_compute_global_address.default.address : var.ip_address}"
```

Thanks !

**Comment by tpoindessous** [comment]
I signed it !

**Comment by tpoindessous** [comment]
I signed it!

File: main.tf [hunk 3]
```diff
@@ -54,7 +55,7 @@ resource "google_compute_target_https_proxy" "default" {
   count            = "${var.ssl ? 1 : 0}"
   name             = "${var.name}-https-proxy"
   url_map          = "${element(compact(concat(list(var.url_map), google_compute_url_map.default.*.self_link)), 0)}"
-  ssl_certificates = ["${compact(concat(var.ssl_certificates, google_compute_ssl_certificate.default.*.self_link))}"]
+  ssl_certificates = ["${var.certificate}"]
```
Comment (tpoindessous) [inline main.tf]: I think this is a typo ...

correct original code : 
```
ssl_certificates = ["${compact(concat(var.ssl_certificates, google_compute_ssl_certificate.default.*.self_link))}"]
```

**Pull request closed without merging**